README.md

```diff
@@ -68,24 +68,24 @@ w3 up recipies.txt
 
```

Comment:
Please can you remove "coming soon!" from the ToC?

Reply:
done